api/src/main/java/org/infinispan/api/sync/SyncList.java

```diff
@@ -0,0 +1,27 @@
+package org.infinispan.api.sync;
+
+import java.util.Collection;
+
+/**
+ * @since 15.0
+ **/
+public interface SyncList<V> {
+
+   String name();
+
+   SyncContainer container();
+
+   void offerLast(V value);
+
+   void offerLast(V value, long count);
+
+   void offerFirst(V value);
+
+   void offerFirst(V value, long count);
```

Comment:
What's the significance of the `long` here?

Reply:
offer N values atomically. this is a redis list op, not important for this pr